requirements/base.txt

```diff
@@ -1,5 +1,5 @@
 boto3==1.10.8
-django==1.11.23
+django~=2.2
```

Comment:
I would prefer to pin to exact version. I like looser pinning when there's a manifest and a lock, but when there's only one file I think it's best to be exact.